src/bosh_openstack_cpi/lib/cloud/openstack/cloud.rb

```diff
@@ -149,11 +166,11 @@ def create_stemcell(image_path, cloud_properties)
     # @return [Hash] normalized properties
     def normalize_image_properties(properties)
       image_properties = {}
-      image_options = ['name', 'version', 'os_type', 'os_distro', 'architecture', 'auto_disk_config',
+      image_options = ['version', 'os_type', 'os_distro', 'architecture', 'auto_disk_config',
                        'hw_vif_model', 'hypervisor_type', 'vmware_adaptertype', 'vmware_disktype',
                        'vmware_linked_clone', 'vmware_ostype']
       image_options.reject { |image_option| properties[property_option_for_image_option(image_option)].nil? }.each do |image_option|
-        image_properties[image_option.to_sym] = properties[property_option_for_image_option(image_option)]
+        image_properties[image_option.to_sym] = properties[property_option_for_image_option(image_option)].to_s
```

Comment:
Image V2 seems to require that all Custom Properties have a string value. Removing this conversion throws this error from the API:

```
400 Bad Request\n\nProvided object does not match schema 'image': True is not of type 'string'  Failed validating 'type' in schema['additionalProperties']:     {'type': 'string'}  On instance[u'auto_disk_config']:     True\n\n   "
```